### PR TITLE
Added Swap odd even bits algoritm

### PR DESCRIPTION
**Pull request by somya51p**
@tarun26091999 Added the swap all odd and even bits of a number algorithm in **C++.**
This PR is associated to issue number #1201.
Please review the PR and suggest changes if any. Thank You!

### CODE REVIEW

**Comment by anishsingh42** [comment]
@tarun26091999 @plazzy99 Would you please assign me this issue.
Language :- Python.

File: Code/C++/Swap_odd_even_bits.cpp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    printf("%u ", swap_oddeven_Bits(x)); 
+  
+    return 0; 
+} 
+
+
+/*  Input:- 23
+    Output:- 43
+    where, 23 is (00010111)
+    and    43 is (00101011)
```
Comment (rish-singhal) [inline Code/C++/Swap_odd_even_bits.cpp:43]: add space and time complexity

File: Code/C++/Swap_odd_even_bits.cpp [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    return (even_bits | odd_bits); // Combine even and odd bits 
+} 
+  
+int main() 
+{ 
+    unsigned int x;
+    cin>> x;
+ 
+    printf("%u ", swap_oddeven_Bits(x)); 
+
```
Comment (rish-singhal) [inline Code/C++/Swap_odd_even_bits.cpp:35]: remove extra line

File: Code/C++/Swap_odd_even_bits.cpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+*/
+
+
+#include <bits/stdc++.h>
+using namespace std;
+  
+// Function to swap even and odd bits 
+unsigned int swap_oddeven_Bits(unsigned int x) 
+{ 
+    // Get all even bits of x
```
Comment (rish-singhal) [inline Code/C++/Swap_odd_even_bits.cpp:17]: explain a bit, or reference something~ about the working of this

File: Code/C++/Swap_odd_even_bits.cpp [hunk 4]
```diff
@@ -0,0 +1,44 @@
+/* Swap all odd and even bits of a given number
+
+    This algoritm would use Bit Manipulation.
+    If we take a closer look, we can observe that we basically need to right shift (>>) all even bits by 
+    1 so that they become odd bits, and left shift (<<) all odd bits by 1 so that they become even bits.
+
+    For example, x = 23 (00010111) with output 43 (00101011)
+*/
+
```
Comment (rish-singhal) [inline Code/C++/Swap_odd_even_bits.cpp:9]: remove extra line

File: Code/C++/Swap_odd_even_bits.cpp [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    cin>> x;
+ 
+    printf("%u ", swap_oddeven_Bits(x)); 
+  
+    return 0; 
+} 
+
+
+/*  Input:- 23
+    Output:- 43
```
Comment (rish-singhal) [inline Code/C++/Swap_odd_even_bits.cpp:41]: add at least 2 test cases?

**Comment by rish-singhal** [comment]
Also, please update read me accordingly to resolve conflicts.

**Pull request closed without merging**